test/examples_test.go

```diff
@@ -65,18 +69,12 @@ func TestTerraform_EC2DemoExample(t *testing.T) {
 		return false
 	}, 2*time.Minute, 10*time.Second)
 
-	var clients struct {
-		CountingURL  string `json:"counting_url"`
-		DashboardURL string `json:"dashboard_url"`
-	}
-	terraform.OutputStruct(t, terraformOptions, "clients", &clients)
-
-	resp, err := http.Get(clients.CountingURL)
+	resp, err := http.Get(terraform.Output(t, terraformOptions, "nomad"))
```

Comment:
I think the `nomad` and `hashicups` ouputs were updated with a `_url` suffix after these tests were updated!